community/dbms/src/main/java/org/neo4j/commandline/dbms/ImportCommand.java

```diff
@@ -41,10 +41,15 @@
 import org.neo4j.kernel.impl.util.Validators;
 import org.neo4j.server.configuration.ConfigLoader;
 
+import static org.neo4j.csv.reader.Configuration.DEFAULT;
+import static org.neo4j.unsafe.impl.batchimport.Configuration.DEFAULT_MAX_MEMORY_PERCENT;
+import static org.neo4j.unsafe.impl.batchimport.input.csv.Configuration.COMMAS;
+
 public class ImportCommand implements AdminCommand
 {
     public static final String DEFAULT_REPORT_FILE_NAME = "import.report";
     private static final String[] allowedModes = {"database", "csv"};
+    private static final String[] allowedDelimiterCharacters = {",", "TAB", ";", "|", ":", "#"};
```

Comment:
But this is not new; `\` would always have conflicted. But by restricting to a specific set of characters you are setting yourself up to get a feature request to add X to the list of allowed characters, or complaints that "This used to work in import-tool". I just think it's not worth that hassle without good reason.